3ds Max/Max2Babylon/Forms/AnimationGroupControl.cs

```diff
@@ -167,6 +170,40 @@ private void confirmButton_Click(object sender, EventArgs e)
             ConfirmPressed?.Invoke(confirmedInfo);
         }
 
+        private AnimationGroup CalculateEndFrameFromAnimationGroupNodes(AnimationGroup animationGroup)
```

Comment:
This should probably return int, rather than AnimationGroup if we ever want to make this reusable.

Reply:
Also, why put this in AnimationGroupControl, rather than as part of [AnimationGroup](https://github.com/BabylonJS/Exporters/blob/master/3ds%20Max/Max2Babylon/Exporter/AnimationGroup.cs) as a utility or class method?

Reply:
ok I will move this to utility